strict_deps/index.mts

```diff
@@ -1,52 +1,58 @@
-import {StrictDepsManifest} from './manifest.mjs';
-import fs from 'fs/promises';
+import fs from 'node:fs/promises';
+import path from 'node:path';
 import ts from 'typescript';
-import {getImportsInSourceFile} from './visitor.mjs';
 import {createDiagnostic} from './diagnostic.mjs';
-import path from 'path';
+import {StrictDepsManifest} from './manifest.mjs';
+import {getImportsInSourceFile} from './visitor.mjs';
 
 const [manifestExecPath, expectedFailureRaw] = process.argv.slice(2);
 const expectedFailure = expectedFailureRaw === 'true';
 
 const manifest: StrictDepsManifest = JSON.parse(await fs.readFile(manifestExecPath, 'utf8'));
 
 const extensionRemoveRegex = /\.[mc]?(js|ts)$/;
+const outputFileRemoveRegex = /^bazel-out\/k8-fastbuild\/bin\//;
```

Comment:
This won't work as the `k8-fastbuild` portion could be anything. Should we instead simply pass `.short_path`'s to the manifest? That shouldn't include these fragments